00-programming-fundamentals/README.md

```diff
@@ -0,0 +1,30 @@
+## Programming Fundamentals
+
+
+### Introductory
+1. [How To OSX](01-how-to-osx.md)
+1. [OSX Useful Apps](02-osx-useful-apps.md)
+1. [Intro to the Command Line](03-intro-to-the-command-line.md)
+1. [Installfest](04-installfest.md)
+1. [Get to know your Text Editor](05-get-to-know-your-text-editor.md)
+
+
+### Process
+1. [What is a retrospective?](06-what-is-a-retrospective.md)
+1. [Introduction to Agile](12-intro-to-agile.md)
+1. [Pair Programming](pair-programming.md)
+1. [What is Pseudocode?](pseudocode.md)
+
+### Tools
+1. [Introduction to Automated Tests](08-intro-to-automated-tests.md)
+1. [Debugging with Pry](debugging-with-pry.md)
+1. [Code Coverage & SimpleCov](16-code-coverage-and-simplecov.md)
+
+### Git
+1. [Submitting Projects](07-submitting-projects-v1.md)
+1. [Managing Git branches](13-managing-git-branches.md)
+1. [Submit a Project using Branches](15-git-submit-a-project.md)
```

Comment:
I honestly wasn't sure if we used it or not - I still like the idea of them submitting the baseline PR for Rails projects and us merging it in so we don't have to wade through the hundreds of rails boilerplate files....